pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3817,8 +3817,10 @@
         "4.0.2.0": "4.3.0"
       }
     },
-    "System.Runtime.Intrinsics.Experimental": {
-      "InboxOn": {},
+    "System.Runtime.Intrinsics": {
+      "InboxOn": {
+        "netcoreapp3.0": "4.0.1.0",
+      },
       "AssemblyVersionInPackageVersion": {
```

Comment:
I *think* `AssemblyVersionInPackageVersion` can be removed since we no longer make this package. @weshaggard @ericstj - thoughts?

Reply:
I kept it here, right now, since it existed before the original change in 7e5f966

Reply:
Since there is no package you should remove this section.

Reply:
Fixed.